docs/user/whats-new.asciidoc

```diff
@@ -5,12 +5,254 @@ Here are the highlights of what's new and improved in {minor-version}.
+
+* *Easy start:* To begin using {esql} in Discover, select “Try {esql}” from the data-view picker.
+* *Efficient and easy query building:* {esql} in Discover offers auto-complete and in-app documentation, making it easy to craft powerful queries right from the query bar. 
+* *Comprehensive and powerful data exploration:* Conduct ad-hoc data exploration within Discover. Create aggregations, transform data, enrich datasets, and more directly from the query builder. Results are presented in a tabular format or as visualizations. It depends on the query you are executing. 
+* *Contextual Visualizations:* When writing {esql} queries in Discover, you’ll receive visual representations powered by the Lens suggestion engine. Your query’s nature determines the type of visualization you get, such as a metric, histogram, heatmap, and so on. 
+* *Enrichment:* Use the enrich command to enhance your query dataset with fields from another dataset, complete with in-context suggestions for the selected policy, such as hinting the matching field and enriched columns.
+* *In-line visualization editing:* Edit {esql} visualizations directly within Discover and Dashboards. No need to navigate to Lens for quick edits, so you can make changes seamlessly. 
+* *Dashboard integration:* Save your {esql} visualizations to a Dashboard directly from Discover once you’re satisfied with the results. 
+* *Alerting:* Use {esql} for observability and security alerts, setting aggregated values as thresholds. Enhance detection accuracy and receive actionable notifications by emphasizing meaningful trends over isolated incidents, reducing false positives. 
+
+{esql} in Discover brings efficiency and power to your data investigations, streamlining your path to insights. 
+
+Learn more about {esql}’s capabilities in the link:{ref-bare}/master/esql.html[{es} {esql} documentation]. 
+
+.Example of an {esql} query with an {observability} use case: 
+
+[role="screenshot"]
+image::images/obvs.png[An example of {esql}]
+
+[role="screenshot"]
+image::images/obv-use-case.png[An example using {esql} in an {observability} use case]
+
+.Example of {esql} query with {observability} use case and using grok: 
+
+[role="screenshot"]
+image::images/grok.png[An example of {esql}]
+
+[role="screenshot"]
+image::images/grok-use-case.gif[An example using {esql} in an {observability} use case]
+
+[discrete]
+=== Dashboard 
+
+[discrete]
+==== Lens inline editing in Dashboards
+
+You can now edit a **Lens** visualization without leaving the dashboard instead of navigating back and forth to the Lens editor. In the upper right nn the visualization panel, click **&#8230;  > Edit visualization**. A flyout will be open in the dashboard where you can perform any edits to your **Lens** panels. Once happy with your edits, click **Apply and close**. This new editing experience is more convenient and will save you time since the dashboard will not need to reload when saving your changes. 
```

Comment:
Avoid using terms like "upper right" for accessibility reasons.  Also, this menu is called the panel menu.

```suggestion
You can now edit a **Lens** visualization without leaving the dashboard instead of navigating back and forth to the Lens editor. Open the panel menu and select **Edit visualization**. A flyout will be open in the dashboard where you can perform any edits to your **Lens** panels. Once happy with your edits, click **Apply and close**. This new editing experience is more convenient and will save you time since the dashboard will not need to reload when saving your changes. 
```